apps/shared/avifjpeg.c

```diff
@@ -50,7 +50,7 @@ avifBool avifJPEGRead(avifImage * avif, const char * inputFilename, avifPixelFor
     FILE * f = fopen(inputFilename, "rb");
     if (!f) {
         fprintf(stderr, "Can't open JPEG file for read: %s\n", inputFilename);
-        goto cleanup;
+        return ret;
```

Comment:
We may be able to shuffle the code so that cinfo is declared and initialized before we call fopen(), but I was too lazy to look into that.